<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nivel</class>
 <widget class="QDialog" name="Nivel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Escolha a quantidade de discos:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>83</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radio3">
      <property name="text">
       <string>3 discos</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radio4">
      <property name="text">
       <string>4 discos</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="radio5">
      <property name="text">
       <string>5 discos</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="radio6">
      <property name="text">
       <string>6 discos</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="radio7">
      <property name="text">
       <string>7 discos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
